lib/modules/datasource/docker/index.ts

```diff
@@ -939,7 +939,7 @@ export class DockerDatasource extends Datasource {
    * This function will filter only tags that contain a semver version
    */
   @cache({
-    namespace: 'datasource-docker-releases',
+    namespace: 'datasource-docker-releases-v2',
```

Comment:
why this change? 🤔 cache will invalidate anyways after 15 min

Reply:
Fallback cache is longer, though